app/src/androidTest/java/org/projectbuendia/client/ui/lists/LocationListActivityTest.java

```diff
@@ -55,31 +55,33 @@ public void testZonesAndTentsDisplayed_afterSettingsView() {
 
     /** Tests that zones and tents are still displayed after returning from chart view. */
     public void testZonesAndTentsDisplayed_afterChartView() {
-        inUserLoginInitDemoPatient();
-        inUserLoginGoToLocationSelection();
-        inLocationSelectionClickLocation(LOCATION_NAME);
-        inPatientListClickFirstPatient(); // open patient chart
-
-        pressBack(); // back to search fragment
-        pressBack(); // back to tent selection screen
+        inUserLoginGoToPatientCreation();
+        screenshot("Test Start");
+        String id = generateId();
```

Comment:
you don`t need id in any other place, generate it inside populateNewPatientFields()